docs/release-notes/release-notes-1.7.0.md

```diff
@@ -0,0 +1,1434 @@
+even if you don't intend to vote in favor of the agenda.**
+
+## Downgrade Warning
+
+The database format in v1.7.0 is not compatible with previous versions of the
+software.  This only affects downgrades as users upgrading from previous
+versions will see a one time database migration.
+
+Once this migration has been completed, it will no longer be possible to
+downgrade to a previous version of the software without having to delete the
+database and redownload the chain.
+
+The database migration typically takes around 40-50 minutes on HDDs and 20-30
+minutes on SSDs.
+
+## Notable Changes
+
+### Four New Consensus Change Votes
+
+Four new consensus change votes are now available as of this release.  After
+upgrading, stakeholders may set their preferences through their wallet.
+
+### Revert Treasury Maximum Expenditure Policy Vote
+
+The first new vote available as of this release has the id `reverttreasurypolicy`.
+
+The primary goal of this change is to revert the currently active maximum
+expenditure policy of the decentralized Treasury to the one specified in the
+[original Politeia proposal](https://proposals-archive.decred.org/proposals/c96290a).
+
+See [DCP0007](https://github.com/decred/dcps/blob/master/dcp-0007/dcp-0007.mediawiki) for
+the full technical specification.
+
+### Explicit Version Upgrades Vote
+
+The second new vote available as of this release has the id `explicitverupgrades`.
+
+The primary goals of this change are to:
+
+* Provide an easy, reliable, and efficient method for software and hardware to
+  determine exactly which rules should be applied to transaction and script
+  versions
+* Further embrace the increased security, and other desirable properties, that
+  hard forks provide over soft forks
+
+See the following for more details:
+
+* [Politeia proposal](https://proposals.decred.org/record/3a98861)
+* [DCP0008](https://github.com/decred/dcps/blob/master/dcp-0008/dcp-0008.mediawiki)
+
+### Automatic Ticket Revocations Vote
+
+The third new vote available as of this release has the id `autorevocations`.
+
+The primary goals of this change are to:
+
+* Improve the Decred stakeholder user experience by removing the requirement for
+  stakeholders to manually revoke missed and expired tickets
+* Enable the recovery of funds for users who lost their redeem script for the
+  legacy VSP system (before the release of vspd, which removed the need for the
+  redeem script)
+
+See the following for more details:
+
+* [Politeia proposal](https://proposals.decred.org/record/e2d7b7d)
+* [DCP0009](https://github.com/decred/dcps/blob/master/dcp-0009/dcp-0009.mediawiki)
+
+### Change PoW/PoS Subsidy Split to 10/80 Vote
+
+The fourth new vote available as of this release has the id `changesubsidysplit`.
+
+The proposed modification to the subsidy split is intended to substantially
+diminish the ability to attack Decred's markets with mined coins and improve
```

Comment:
Mentioning markets here doesn't sit right with me. This is a software release document and IMO should be focused on the technical aspects of the release - changes to the software, new features etc.

I'd keep this section focused on the nature of the proposed consensus change and not mention market/attack theories. There is a link to the proposal just below this for those who are interested in reading further justification for the change.

Reply:
This section serves to quickly highlight the motivation for changes at a quick glance as opposed to technically describe them.  Take a look at all of the other consensus changes in this section and they are all "The primary goal(s) of this change...." and/or "The intention of this ...".

This description is directly from the motivation section of the DCP, so I don't see any reason to avoid mentioning it here.